service/src/main/java/bio/terra/workspace/service/workspace/flight/RemoveUserFromWorkspaceFlight.java

```diff
@@ -24,8 +24,11 @@ public RemoveUserFromWorkspaceFlight(FlightMap inputParameters, Object applicati
         inputParameters.get(JobMapKeys.AUTH_USER_INFO.getKeyName(), AuthenticatedUserRequest.class);
     String userToRemove = inputParameters.get(WorkspaceFlightMapKeys.USER_TO_REMOVE, String.class);
     Optional<WsmIamRole> roleToRemove =
-        Optional.ofNullable(
-            inputParameters.get(WorkspaceFlightMapKeys.ROLE_TO_REMOVE, WsmIamRole.class));
+        inputParameters.get(WorkspaceFlightMapKeys.ROLE_TO_REMOVE, String.class) != null
```

Comment:
I'm not seeing why you switched from WsmIamRole to the role name string. Role coming into WorkspaceService has to be a non-null, valid WsmIamRole, right?

Reply:
If flight key is WsmIamRole, then RemoveUserFromWorkspaceFlightTest fails with JsonConversionException

https://scans.gradle.com/s/nmjwrcsi7efm2/tests/:service:connectedTest/bio.terra.workspace.service.workspace.flight.RemoveUserFromWorkspaceFlightTest/removeUserFromWorkspaceFlightDoUndo()?top-execution=1

```
<div class="FailureException__stacktrace-summary StacktraceSummary" style="box-sizing: border-box; margin-top: 8px; align-items: stretch; color: rgb(104, 104, 104); display: flex; font: 400 14px Inconsolata, Monaco, &quot;Courier New&quot;, monospace; position: relative; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><br class="Apple-interchange-newline">

  |   | at bio.terra.stairway.impl.FlightFactory.makeFlight(FlightFactory.java:35) |  
-- | -- | -- | --
  |   | •••
  |   | at bio.terra.workspace.common.StairwayTestUtils.blockUntilFlightCompletes(StairwayTestUtils.java:42) |  
  |   | at bio.terra.workspace.service.workspace.flight.RemoveUserFromWorkspaceFlightTest.removeUserFromWorkspaceFlightDoUndo(RemoveUserFromWorkspaceFlightTest.java:171) |  
  |   | •••

</div></div></div></div></div></div></div>
```

Other enum flight keys are also stored as string, I assume for similar reason. Eg

https://cs.github.com/DataBiosphere/terra-workspace-manager/blob/db7746b1fb275333dc8747ca7e9bdb0bda8af87e/service/src/main/java/bio/terra/workspace/service/workspace/flight/WorkspaceDeleteFlight.java?q=WorkspaceFlightMapKeys.WORKSPACE_STAGE#L25